spec/controllers/users_controller_spec.rb

```diff
@@ -58,4 +87,17 @@
       end
     end
   end
+
+  context "DELETE #destroy" do
+    context "when not logged in" do
+      it "won't delete the user" do
+        user = create(:user)
+
+        delete :destroy, params: { id: user.id }
+
+        expect(flash[:alert]).
+          to match I18n.t("sessions.new.logged_in")
+      end
+    end
+  end
```

Comment:
Since you are checking that the before filter works in other actions, I would probably won't bother here. 
But if you decide to keep this spec, like before, there is no need to actually create a user here, because the `before_action` should be triggered before the `find` user call

Reply:
Done in sha: 08e222e